resources/serverless/values.yaml

```diff
@@ -58,12 +58,11 @@ tests:
 global:
   domainName: "kyma.example.com"
   commonLabels:
-    app: '{{ template "name" . }}'
-    version: "{{ .Values.global.images.function_controller.version }}"
-    app.kubernetes.io/name: '{{ template "name" . }}'
+    app: "{{ .Release.Name }}"
+    app.kubernetes.io/name: "{{ .Release.Name }}"
     app.kubernetes.io/instance: "{{ .Release.Name }}"
+    app.kubernetes.io/component: '{{ template "fullname" . }}-controller'
```

Comment:
What about just `controller`?

Reply:
I think that might collide with other components that could be using the same pattern. This way it's more specific.

Reply:
I wouldn't say it collide.
If I want to get all k8s controllers, I am able to do that with that label.

Reply:
Look at the docs: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels
The component is named just: `database`

Reply:
Controller deployment template has the `kyma-project.io/component: controller` which would suitable for matching controllers. Using the `app.kubernetes.io/component` label this way is more aligned with the kubernetes docs recommendation.